### DISCUSSION TITLE
[Solana] Phantom selection failing in iOS through WC modal

### DISCUSSION THREAD

**Discussion by losman0s** [WalletConnect/walletconnect-monorepo#3744] [Developer Support] [answered]
Hi, Solana-specific question:

I am trying to enable Phantom usage in my iOS PWA, through the WC modal.
At first Phantom (which appears supported in your explorer) was not listed as an option at all.
After specifying its ID explicitly in the `explorerRecommendedWalletIds` option (`a797aa35c0fadbfc1a53e7f675162ed5226968b44a19ee3d24385c64d1d3c393`), it appeared but shows "Not Detected" when selected, despite the app being installed on the device.

Configs for the modal:

```ts
const modal = new WalletConnectModal({
  projectId: "xxx",
  chains: ["solana:4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ"],
  explorerRecommendedWalletIds: ["a797aa35c0fadbfc1a53e7f675162ed5226968b44a19ee3d24385c64d1d3c393"]
});
```

Modal gets opened without particular option (`modal.openModal();`).

Any idea what could be wrong?

**Comment by losman0s** [comment]
cc @boidushya as mentioned by Glitch in Discord

**Reply by boidushya** [reply]
Right now, they don't have a Deep Link for mobile or desktop in their submission. Thanks for bringing this up, will raise this with the Phantom team 🤝

**Reply by boidushya** [reply]
Update: Phantom doesn't seem to support all of WalletConnect's functionalities just yet. Meanwhile you could use https://www.spot-wallet.com/ for this.

**Reply by losman0s** [reply]
Is that what is missing on your side? https://docs.phantom.app/phantom-deeplinks/deeplinks-ios-and-android

**Reply by glitch-txs** [reply, marked as answer]
nope, they need to implement WalletConnect protocol which is not supported by Phantom atm
